addons/binding/org.openhab.binding.sinope/src/main/java/org/openhab/binding/sinope/handler/SinopeThermostatHandler.java

```diff
@@ -11,10 +11,10 @@
 import java.io.IOException;
 import java.net.UnknownHostException;
 
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
Please do not use `@NonNull`. See https://www.eclipse.org/smarthome/documentation/development/guidelines.html#a-code-style point 12. If you want to use these annotations, please use `@NonNullbyDefault` and `@Nullable`